release-notes/System.ServiceModel.Primitives.md

```diff
@@ -0,0 +1,15 @@
+# System.ServiceModel.Primitives
+
+## Information
+
+* [NuGet Package](https://nuget.org/packages/System.ServiceModel.Primitives)
+* [Report an issue](http://github.com/dotnet/wcf/issues/new)
+* [Source](https://github.com/dotnet/wcf/tree/master/src)
+
+## Version History
+
+### 4.1.0-beta-23516 
+
+* [Release tag](https://github.com/dotnet/wcf/tree/v1.0.0-rc1)
+* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v1.0.0-rc1.md)
+* Adds new public API surface area to enable use of X509Certificate2
```

Comment:
I agree we should not have a System.Private.ServiceModel.md. I am a little concerned useful information will be fragmented by keeping multiple .md's. WCF components are coupled much more together, which is different than components in corefx repro. So I vote for having "what's new" in central release notes. For RC1, yes, it will be new from beta8.